specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-03-01-preview/WorkspaceManagerMembers.json

```diff
@@ -0,0 +1,304 @@
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete a workspace manager member": {
+            "$ref": "./examples/workspaceManagerMembers/DeleteWorkspaceManagerMember.json"
+          }
+        },
+        "tags": [
+          "workspaceManagerMember"
+        ],
+        "description": "Deletes a workspace manager member",
+        "operationId": "WorkspaceManagerMembers_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceManagerMemberName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "WorkspaceManagerMemberProperties": {
+      "description": "The workspace manager member properties",
+      "properties": {
+        "targetWorkspaceId": {
+          "description": "Fully qualified resource ID of the target Sentinel workspace joining the given Sentinel workspace manager",
+          "type": "string"
+        }
+      },
+      "required": [
+        "targetWorkspaceId"
+      ],
+      "type": "object"
+    },
+    "WorkspaceManagerMember": {
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/AzureEntityResource"
+        }
+      ],
+      "description": "The workspace manager member",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/WorkspaceManagerMemberProperties",
+          "description": "The workspace manager member object",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "WorkspaceManagerMembersList": {
+      "description": "List of workspace manager members",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "URL to fetch the next set of workspace manager members",
+          "readOnly": true,
```

Comment:
>readOnly [](http://example.com/codeflow?start=11&length=8)

Q for SDK folks. Are we supposed to use format uri on these? Not sure if it matters.